openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -506,7 +506,7 @@ describe("OpenAssessment.ResponseView", function() {
         var files = [{type: 'image/jpeg', size: 22000000, name: 'huge-picture.jpg', data: ''}];
         view.prepareUpload(files, 'image');
         expect(view.baseView.toggleActionError).toHaveBeenCalledWith('upload',
-            'File size must be 20MB or less.');
+            'Total file upload size must be 20MB or less.');
```

Comment:
I think this case is no longer relevant.

Reply:
Yeah, you're right. I'll also have to change some of the other validation